app/main/forms.py

```diff
@@ -894,7 +894,7 @@ class SupportType(StripWhitespaceForm):
 
 class Feedback(StripWhitespaceForm):
     name = StringField('Name')
```

Comment:
```suggestion
    name = StringField('Name (optional)')
```

> If you ask for optional information, mark the labels of optional fields with ‘(optional)’.
– https://design-system.service.gov.uk/patterns/question-pages/#asking-users-questions

Reply:
Yep, have added this change. It also does it on both pages already.